services-gateway-websocket/src/main/java/io/scalecube/services/gateway/ws/WebsocketGatewayAcceptor.java

```diff
@@ -52,25 +52,21 @@ public WebsocketGatewayAcceptor(ServiceCall serviceCall, GatewayMetrics metrics)
 
     session
         .receive()
+        .map(messageCodec::decode)
```

Comment:
so if something will happen with decoding or the sender won't send `sid` - we will have an exception and the whole connection will be closed, we will not receive anything from this stream!